dm/pkg/conn/db.go

```diff
@@ -132,48 +135,21 @@ func GetSlaveServerID(ctx context.Context, db *sql.DB) (map[uint32]struct{}, err
 	var rowsResult []string
 	rowsResult, err = export.GetSpecifiedColumnValueAndClose(rows, "Server_id")
 	if err != nil {
-		return nil, terror.DBErrorAdapt(err, terror.ErrDBDriverError)
+		return nil, terror.DBErrorAdapt(err, db.Scope, terror.ErrDBDriverError)
 	}
 	for _, serverID := range rowsResult {
 		// serverID will not be null
 		serverIDUInt, err := strconv.ParseUint(serverID, 10, 32)
 		if err != nil {
-			return nil, terror.DBErrorAdapt(err, terror.ErrDBDriverError)
+			return nil, terror.DBErrorAdapt(err, db.Scope, terror.ErrDBDriverError)
 		}
 		serverIDs[uint32(serverIDUInt)] = struct{}{}
 	}
 	return serverIDs, nil
 }
 
-// GetGlobalVariable gets server's global variable.
-func GetGlobalVariable(ctx context.Context, db *sql.DB, variable string) (value string, err error) {
```

Comment:
this code duplicate with code in `conn/utils.go`, only paramater is different, so it's deleted, below `getVariable` too